stylesheets/styles.css

```diff
@@ -739,6 +739,7 @@ input {
   height: 100vh;
   z-index: 9999;
   background: #6070ff;
+  opacity: 0.93;
```

Comment:
- [ ] As requested by the previous reviewer, please consider fixing the style for your mobile menu interface. 

|Template|Your design|
|---|---|
| ![image](https://github.com/hafedEfheij/PortfolioPoject/assets/77069011/d697efd8-c0ef-42d2-869b-8776d291ab55) | ![image](https://github.com/hafedEfheij/PortfolioPoject/assets/77069011/6c6d16fb-bce8-45b6-be95-5306e269267c) |

The text shouldn't have this opacity effect, please consider fixing this! 🤓 🔧